pkg/tagger/collectors/kubelet_extract.go

```diff
@@ -85,6 +85,9 @@ func (c *KubeletCollector) parsePods(pods []*kubelet.Pod) ([]*TagInfo, error) {
 		// Pod phase
 		tags.AddLow("pod_phase", strings.ToLower(pod.Status.Phase))
 
+		// Priority class
+		tags.AddLow("kube_priority_class", pod.Spec.PriorityClassName)
```

Comment:
we can probably edit a unit test case to test extracting this field

Reply:
Test case added.